modules/ROOT/pages/deprecations-additions-removals-compatibility.adoc

```diff
@@ -47,6 +47,29 @@ It returns a string representation of the property type for property type constr
 | Feature
 | Details
 
+a|
+label:functionality[]
+label:new[]
+[source, cypher, role="noheader"]
+----
+IS [NOT] :: <TYPE>
+----
+a|
+
+Added type predicate expressions. The available types are:
```

Comment:
Should this be linked to the docs about them for easy access?

Reply:
should also be added to the one for constraints then (and I know this is one of the lines I've updated in my PR though not to add a link)